--- bindings_orig.go	2021-01-04 20:39:18.000000000 +0900
+++ bindings.go	2021-01-04 20:17:14.000000000 +0900
@@ -25,7 +25,7 @@
 // NOTE: On mobile platforms the '--lnddir` argument should be set to the
 // current app directory in order to ensure lnd has the permissions needed to
 // write to it.
-func Start(extraArgs string, unlockerReady, rpcReady Callback) {
+func Start(extraArgs string, unlockerReady Callback) {
 	// Split the argument string on "--" to get separated command line
 	// arguments.
 	var splitArgs []string
@@ -63,7 +63,6 @@
 	// to accept calls.
 	var (
 		unlockerListening = make(chan struct{})
-		rpcListening      = make(chan struct{})
 	)
 
 	// We call the main method with the custom in-memory listeners called
@@ -73,10 +72,6 @@
 			Listener: walletUnlockerLis,
 			Ready:    unlockerListening,
 		},
-		RPCListener: &lnd.ListenerWithSignal{
-			Listener: lightningLis,
-			Ready:    rpcListening,
-		},
 	}
 
 	// Call the "real" main in a nested manner so the defers will properly
@@ -112,20 +107,2 @@
 
 		unlockerReady.OnResponse([]byte{})
- 	}()
-
-	go func() {
-		<-rpcListening
-
-		// Now that the RPC server is ready, we can get the needed
-		// authentication options, and add them to the global dial
-		// options.
-		auth, err := lnd.AdminAuthOptions(loadedConfig)
-		if err != nil {
-			rpcReady.OnError(err)
-			return
-		}
-
-		// Add the auth options to the listener's dial options.
-		addLightningLisDialOption(auth...)
-
-		rpcReady.OnResponse([]byte{})
	}()
 }
\ No newline at end of file
